### DISCUSSION TITLE
Water Flow Meter FL-808

### DISCUSSION THREAD

**Discussion by boilermanc** [arendst/Tasmota#13485] [Q&A] [answered]
I am connecting a Digiten Flow Meter FL-808 to a wemos d1 r2. I have the yellow wire connected to the d2 gpio on the wemos. Think thats correct from what I have been able to research. Not sure what to set the gpio to in Tasmota? Would this sensor be supported?

thanks!

**Comment by sfromis** [comment, marked as answer]
Since it appears to have pulses as output, you can use a gpio config of Counter to have Tasmota count those pulses. Getting from pulses to quantity is up to you to manage. This generic support means that Tasmota does not "know" details of the wide array of supportable rate sensors.

Take care to use the correct 3.3V signal voltage which is what ESP8266 supports.

**Comment by boilermanc** [comment]
Thanks for that. Which counter would you recommend. There are several of them.

Thanks!

⁣Get BlueMail for Android ​

**Reply by sfromis** [reply]
Counter_n is the same component, just without enabling the internal pullup resistor. Since it is a hall effect sensor, the internal pullup may or may not be needed. The numbered dropdown to the right is just if you have more than one separate counter input.
